This patch should fix gentoo bug #99413. This problem is also described at
http://grsecurity.net/pipermail/grsecurity/2006-February/000702.html. Below
is an excerpt from spender's post on the thread:

  On Sun, Feb 05, 2006 at 10:11:01PM -0500, Mike Frysinger wrote:
  > On Sunday 05 February 2006 20:37, General Stone wrote:
  > > The killall5(8) command don't work properly. I don't know why and there
  > > isn't any output in kernel logs.
  > >
  > > As example:
  > > The wdm starts a xconsole with uid/gid 0/0.
  > > When I make 'killall5 -15' or 'killall5 -9' the xconsole won't be killed
  > > :-/.
  > >
  > > Without grsec it works. Any idea?
  >
  > some Gentoo users have mentioned similar issues ...

  Can you try one of the patches in http://grsecurity.net/~spender ?
  After looking at the killall5 source, I believe I've resolved the bug
  (which seems to be due to killall5 being an awful hack of an
  application).

--- linux-2.6.14-grsec-2.1.8-2.6.14.6-200601211647/fs/proc/array.c	2006-02-05 23:18:05.000000000 -0500
+++ linux-2.6.14-grsec-2.1.8-2.6.14.7-200602052251/fs/proc/array.c	2006-02-05 23:18:35.000000000 -0500
@@ -480,8 +480,8 @@ static int do_task_stat(struct task_stru
 		mm ? get_mm_counter(mm, rss) : 0, /* you might want to shift this left 3 */
 	        rsslim,
 #ifdef CONFIG_GRKERNSEC_PROC_MEMMAP
-		PAX_RAND_FLAGS(mm) ? 0 : (mm ? mm->start_code : 0),
-		PAX_RAND_FLAGS(mm) ? 0 : (mm ? mm->end_code : 0),
+		PAX_RAND_FLAGS(mm) ? 1 : (mm ? mm->start_code : 0),
+		PAX_RAND_FLAGS(mm) ? 1 : (mm ? mm->end_code : 0),
 		PAX_RAND_FLAGS(mm) ? 0 : (mm ? mm->start_stack : 0),
 #else
 		mm ? mm->start_code : 0,
